docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -166,6 +170,8 @@ The examples that follow use namelist files ``values1.nml`` and ``values2.nml``,
           Print report of values needed to render template
     --dry-run
           Only log info, making no changes
+    --debug
+          Print all logging messages and the stack trace
```

Comment:
These `.rst` docs will probably merge with conflicts with PR #393, which changes the indentation of a lot of these code blocks. If this PR merges first I'm happy to take care of that on my branch, but thought I should warn you in case mine merges first.

Reply:
It wasn't too bad, should all be corrected